docs/api/portable-stories.md

```diff
@@ -0,0 +1,184 @@
+// but they already contain all decorators from story level, meta level and project level.
+const { Primary, Secondary } = composeStories(stories);
+
+test('renders primary button with default args', () => {
+  render(<Primary />);
+  const buttonElement = screen.getByText('Text coming from args in stories file!');
+  expect(buttonElement).not.toBeNull();
+});
+
+test('renders primary button with overriden props', () => {
+  // You can override props and they will get merged with values from the Story's args
+  render(<Primary>Hello world</Primary>);
+  const buttonElement = screen.getByText(/Hello world/i);
+  expect(buttonElement).not.toBeNull();
+});
+```
+
+### Type
+
+<!-- prettier-ignore-start -->
+```ts
+(
+  csfExports: CSF file exports,
+  projectAnnotations?: ProjectAnnotations
+) => Record<string, ComposedStoryFn>
+```
+<!-- prettier-ignore-end -->
+
+### Parameters
+
+#### `csfExports`
+
+(**Required**)
+
+Type: CSF file exports
+
+Specifies which component's stories you want to compose. Pass the **full set of exports** from the CSF file (not the default export!).
+
+#### `projectAnnotations`
+
+Type: `ProjectAnnotation | ProjectAnnotation[]`
+
+This parameter is provided for convenience. You should likely use [`setProjectAnnotations`](#setprojectannotations) instead. Details about the `ProjectAnnotation` type can be found in that function's [`projectAnnotations`](#projectannotations-2) parameter.
+
+### Return
+
+Type: `Record<string, ComposedStoryFn>`
+
+An object where the keys are the names of the stories and the values are the composed stories.
+
+Additionally, the composed story will have the following properties:
+
+| Property   | Type                                       | Description                                 |
+| ---------- | ------------------------------------------ | ------------------------------------------- |
+| storyName  | `string`                                   | The story's name                            |
+| args       | `Record<string, any>`                      | The story's args                            |
+| argTypes   | `ArgType`                                  | The story's argTypes                        |
+| id         | `string`                                   | The story's id                              |
+| parameters | `Record<string, any>`                      | The story's parameters                      |
+| load       | `() => Promise<void>`                      | Executes all the loaders for a given story  |
+| play       | `(context) => Promise<void>  \| undefined` | Executes the play function of a given story |
+
+## composeStory
+
+You can use `composeStory` if you wish to compose a single story for a component.
+
+```tsx
+// Button.test.ts|tsx
+import { render, screen } from '@testing-library/react';
+import { composeStory } from '@storybook/react';
+import { vi, test, expect } from 'vitest';
+import meta, { Primary as PrimaryStory } from './Button.stories';
```

Comment:
Do you think it's better to show users they should rename the imports in comparison to assigning a different name to their constant e.g.
```ts
import meta, { Primary } from './Button.stories';
const PrimaryStory = composeStory(Primary, meta);
```

Reply:
Good point. I like the more straightforward approach you're suggesting here.